base9/Settings.cpp

```diff
@@ -521,11 +518,10 @@ template<typename T> T Settings::getOrRequest (Node & n, string && f)
     {
         string input = "";
         cout << "Please enter your desired setting for field: '" + f + "':\n";
-        getline(cin, input);
-        cout << "Field '" + f + "' set to: " << input << endl;
```

Comment:
It was for testing.  If there is a problem with the entered value, then I think that is generally reported in the ensuing crash.  This is an interesting design question.  I think one solution might be to, upon successful complete reading of the YAML and retrieval of any missing values via command line input, print to `stdout` a list of all the settings read, or maybe to log this to a file.  That way people could confirm that the settings they put in were correctly read by the code.  But I think that regurgitating values for command-line-entered stuff but not stuff in the YAML file is a sub-optimal solution, and I only had it in there for testing purposes.  So how about you accept this pull request, and then I will make another one with the added feature of optional logging to a log file of the settings read from the YAML and/or command-line input?